site/docs/v1/tech/apis/api-keys.adoc

```diff
@@ -0,0 +1,179 @@
+---
+layout: doc
+title: API Key APIs
+description: APIs for creating, retrieving, updating and deleting API Keys
+---
+
+== Overview
+
+[NOTE.since]
+====
+This API has been available since 1.26.0
+====
+
+The FusionAuth APIs are primarily secured using API keys. This API can only be accessed using an API key that is set as a keyManager. In order to retrieve, update or delete an API key, an API key with equal or greater permissions must be used. A "tenant-scoped" API key can retrieve, create, update or delete an API key for the same tenant. This page describes APIs that are used to manage API keys.
+
+Please refer to the link:/docs/v1/tech/apis/authentication#[Authentication] document for more details about API keys.
+
+* <<Retrieve an API Key>>
+* <<Create an API Key>>
```

Comment:
Looks like the convention is create then retrieve. Can you please switch the links and sections?